app/code/Stanislavz/Chat/view/frontend/web/css/source/_module.less

```diff
@@ -0,0 +1,151 @@
+.chat-clearfix() {
+  &:after {
+    clear: both;
+  }
+}
+
+.hidden {
+  display: none;
+}
+
+.reset-list() {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+& when (@media-common = true) {
```

Comment:
@ahrrhy 
1) Modal has `fixed` position, but is position from the `top` instead of the `bottom` for some reason. Modal  with several messages is under the page bottom if the screen height is not enough.
2) Label must not appear above the textarea. It is not possible to focus the textarea whe clicking the label.